libraries/cloudharness-common/cloudharness/utils/env.py

```diff
@@ -4,28 +4,38 @@
 
 from .. import log
 
+from .config import conf
+
 TEST = 'TEST'
 PROD = 'PROD'
 
 VARIABLE_IMAGE_REGISTRY = 'CH_IMAGE_REGISTRY'
 
 SUFFIX_TAG = 'IMAGE_TAG'
-SUFFIX_PORT = 'PORT'
-SUFFIX_NAME = 'NAME'
+SUFFIX_PORT = 'SERVICE_PORT'
+SUFFIX_NAME = 'SERVICE_HOST'
 
 DEFAULT_IMAGE_REGISTRY = ''
 
 HERE = os.path.dirname(os.path.realpath(__file__))
 
 
 def set_default_environment():
-    with open(HERE + '/resources/values.yaml') as f:
-        values = yaml.safe_load(f)
-        os.environ.update({v['name']: str(v['value']) for v in values['env'] if v['name'] not in os.environ})
+    values = conf.allvalues
+    os.environ.update({v['name']: str(v["value"]) for v in values['env'] if v['name'] not in os.environ})
```

Comment:
I don't really understand the need of this: wouldn't it be easier to get a value from the conf itself rather than creating env variables then looking for the env variable?

Reply:
backwards compatibility